ultra/ultra/train.py

```diff
@@ -272,11 +276,14 @@ def train(
 
         if curriculum_mode is True:
             if CurriculumInfo.static_curriculum_toggle is True:
-                Coordinator.calculate_average_scenario_passed(
-                    episode, total_scenarios_passed, agents, average_scenarios_passed
+                (
+                    average_reached_goal,
+                    total_reached_goal,
+                ) = calculate_average_reached_goal(
+                    episode, total_reached_goal, agents, average_reached_goal
                 )
                 if (episode.index + 1) % CurriculumInfo.pass_based_sample_rate == 0:
-                    print(f"({episode.index + 1}) ASP: {average_scenarios_passed}")
+                    print(f"({episode.index + 1}) ASP: {average_reached_goal}")
```

Comment:
```suggestion
                    print(f"({episode.index + 1}) Average reached goal: {average_reached_goal}")
```
Perhaps change to full name?